cmake/private/FairRootTargets.cmake

```
@@ -67,6 +67,9 @@ function(fairroot_discover_catch2_tests target)
   set(req_version 3.22)
   if(ARGS_ENABLE_LD_LIBRARY_PATH_WORKAROUND)
     set(dl_paths ${LD_LIBRARY_PATH})
+    if(ENV{LD_LIBRARY_PATH})
+      list(APPEND dl_paths $ENV{LD_LIBRARY_PATH})
+    endif()
```

```suggestion
    if(DEFINED ENV{LD_LIBRARY_PATH})
      list(APPEND dl_paths $ENV{LD_LIBRARY_PATH})
    endif()
```
This can fix the issue. Otherwise, the if-statement is always false.